### ISSUE TITLE
conrad show --head and --tail

### ISSUE THREAD

**Issue by vinayak-mehta** [vinayak-mehta/conrad#16]

**Comment by anusha5695** [comment]
@vinayak-mehta : Is this open for PR? if so , any further description?

**Comment by vinayak-mehta** [comment]
@anusha5695 Yes it is open! It should be similar to the unix head and tail commands, to display the first/last n rows of the table.

https://en.wikipedia.org/wiki/Head_(Unix)

**Comment by anusha5695** [comment]
Thanks. I ll pick this up

**Comment by anusha5695** [comment]
Hi @vinayak-mehta, i got it working. Just one question if a user specifies `conrad show --head --tail`, then i presume it will return the head. [ First execute head, then a tail over it will still give the first row . Like tail(head(records)) ]. Is this right?

**Comment by anusha5695** [comment]
@vinayak-mehta : Any updates on this?

**Comment by vinayak-mehta** [comment]
@anusha5695 head and tail are independent options, they shouldn't be called together.

**Comment by anusha5695** [comment]
Okay 👍

**Comment by anusha5695** [comment]
@vinayak-mehta : PR raised
